x-pack/plugins/triggers_actions_ui/public/application/components/health_check.tsx

```diff
@@ -133,46 +166,73 @@ const EncryptionError = ({
+          defaultMessage="You must enable Transport Layer Security"
         />
       </h2>
     }
     body={
       <div className={`${className}__body`}>
         <p role="banner">
-          {i18n.translate(
-            'xpack.triggersActionsUI.components.healthCheck.encryptionErrorBeforeKey',
-            {
-              defaultMessage: 'To create an alert, set a value for ',
-            }
-          )}
-          <EuiCode>{'xpack.encryptedSavedObjects.encryptionKey'}</EuiCode>
-          {i18n.translate(
-            'xpack.triggersActionsUI.components.healthCheck.encryptionErrorAfterKey',
-            {
-              defaultMessage: ' in your kibana.yml file. ',
-            }
-          )}
+          {i18n.translate('xpack.triggersActionsUI.components.healthCheck.tlsError', {
+            defaultMessage:
+              'Alerting relies on API keys, which require TLS between Elasticsearch and Kibana. ',
+          })}
           <EuiLink
-            href={`${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/alert-action-settings-kb.html#general-alert-action-settings`}
+            href={`${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/configuring-tls.html`}
             external
             target="_blank"
           >
-            {i18n.translate(
-              'xpack.triggersActionsUI.components.healthCheck.encryptionErrorAction',
-              {
-                defaultMessage: 'Learn how.',
-              }
-            )}
+            {i18n.translate('xpack.triggersActionsUI.components.healthCheck.tlsErrorAction', {
+              defaultMessage: 'Learn how to enable TLS.',
+            })}
           </EuiLink>
         </p>
       </div>
     }
   />
 );
 
-const TlsError = ({
+const AlertsError = ({
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  docLinks: { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION },
+  className,
+}: PromptErrorProps) => (
+  <EuiEmptyPrompt
+    iconType="watchesApp"
+    data-test-subj="alertsNeededEmptyPrompt"
+    className={className}
+    titleSize="xs"
+    title={
+      <h2>
+        <FormattedMessage
+          id="xpack.triggersActionsUI.components.healthCheck.alertsErrorTitle"
+          defaultMessage="You must enable Alerts"
+        />
+      </h2>
+    }
+    body={
+      <div className={`${className}__body`}>
+        <p role="banner">
+          {i18n.translate('xpack.triggersActionsUI.components.healthCheck.alertsError', {
+            defaultMessage: 'To create an alert, set alerts and actions plugins enabled. ',
+          })}
+          <EuiLink
+            href={`${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/alert-action-settings-kb.html`}
+            external
+            target="_blank"
+          >
+            {i18n.translate('xpack.triggersActionsUI.components.healthCheck.alertsErrorAction', {
+              defaultMessage: 'Learn how to enable Alerts.',
```

Comment:
I notice the docs don't say anything about the `xpack.actions.enabled` setting. Is that an oversight?

Reply:
Yes, this in not documented and we should fix it. In the configs listings for Docker and Cloud we have it.